modules/ROOT/pages/policy-mule4-jwt-validation.adoc

```diff
@@ -155,6 +155,41 @@ Ignore this field if you selected *none* as JWT Signing Method.
 Note that all DataWeave expressions must return a boolean value or they will always fail.
 |===
 
+== Obtaining a policy compatible RSA public key
+The policy uses public rsa key in pem format. In this section we explain how to obtain the public.pem file that contains said key.
+Remember that once the public.pem key is obtained, before pasting it to the policy configuration to remove the *-----BEGIN PUBLIC KEY-----* header, and the *-----END PUBLIC KEY-----* footer.
+
+[%header%autowidth.spread,cols="a,a"]
+|===
+| Starting Point | Procedure
+| No previous certificate |
+
+. Generate the private RSA key
+`openssl genrsa -out key.pem 2048`
+
+. Extract the public key from the generated private key *key.pem*
+`openssl rsa -in key.pem -outform PEM -pubout -out public.pem`
+
+. The public key will be in the file public.pem
+| PEM Certificate
+| . Extract the private key from the pem certificate `openssl x509 -inform pem -in cert.pem -pubkey -noout > public.pem`
+. The public key will be in the file public.pem
+
+| DER Certificate
+| . Extract the private key from the pem certificate `openssl x509 -inform der -in cert.crt -pubkey -noout > public.pem`
+. The public key will be in the file public.pem
+| x5c field from JWKS
+| . Copy the first certificate from the chain and decode it `echo <certificateString> \| base64 -D > cert.crt` (replace `<certificateString>` with the copied value)
+. Extract the private key from the pem certificate `openssl x509 -inform der -in cert.crt -pubkey -noout > public.pem`
+. The public key will be in the file public.pem
+|===
+
+== Accessing propagated claims
```

Comment:
This should be merged with the == Claim Propagation section in line 74 (currently at least)